legal/pekko-actor-jar-license.txt

```diff
@@ -214,3 +214,9 @@ pekko-actor contains code from scala-collection-compat in the `org.apache.pekko.
 which has written by the Scala-Lang team under an Apache 2.0 license.
 Copyright (c) 2002-2023 EPFL
 Copyright (c) 2011-2023 Lightbend, Inc.
+
+---------------
+
+pekko-actor contains code from Netty in `org.apache.pekko.io.dns.DnsSettings.scala`
```

Comment:
INAL, maybe we can add a header in that file?

Reply:
The netty code is Apache licensed so our  file header covers that already. We still need to acknowledge it the main license and notice files.